website/src/pages/playground/useTheme.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { useState, useEffect } from 'react';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+
+function useTheme() {
```

Comment:
Nice hack, I don't think there's a way to listen to theme updates (cc @yangshun in case he knows)

Reply:
This seems to be taken from https://github.com/facebook/docusaurus/blob/master/packages/docusaurus-theme-classic/src/theme/hooks/useTheme.js.

To obtain the theme via context, you can use [ThemeProvider](https://github.com/facebook/docusaurus/blob/master/packages/docusaurus-theme-classic/src/theme/ThemeProvider/index.js). But I guess the intention here is to use a custom implementation.

Reply:
@exced does using ThemeProvider work for you?

Reply:
I wasn't able to import it 😞

Reply:
@rickhanlonii Does it have to be in [exports](https://github.com/facebook/docusaurus/tree/master/packages/docusaurus/src/client/exports) in order to import it ? I didn't manage to import it

Reply:
@exced do this:

```js
import ThemeProvider from '@theme/ThemeProvider';
```

Not `@docusaurus/ThemeProvider`.

Reply:
Hey @yangshun sorry for being late. I tried to use this import and it did not work. Correct me if I'm wrong, `ThemeProvider` is coming from `docusaurus-theme-classic` (which is imported through `docusuraus-preset-classic` in my case) and I can't see it in `node_modules`. I can probably override some stuff with a `theme` folder but I don't think it's the right approach, have you heard of an issue about the server not loading the presets properly ? Thanks

Reply:
Oh that's because I added `ThemeProvider` in a recent commit and it hasn't been published onto npm yet. Give me a while we'll release a new version containing that and you can upgrade to it.

Reply:
`docusaurus@2.0.0-alpha.43` has been released which contains the `ThemeProvider`. Should work now!

Reply:
@yangshun thanks, it works like a charm